certbot/tests/compat/filesystem_test.py

```diff
@@ -353,6 +366,20 @@ def test_check_permissions(self):
             mock_owner.return_value = False
             self.assertFalse(filesystem.check_permissions(self.probe_path, 0o744))
 
+    def test_check_min_permissions(self):
+        filesystem.chmod(self.probe_path, 0o744)
+        self.assertTrue(filesystem.has_min_permissions(self.probe_path, 0o744))
+
+        filesystem.chmod(self.probe_path, 0o700)
+        self.assertFalse(filesystem.has_min_permissions(self.probe_path, 0o744))
+
+    def test_is_world_reachable(self):
+        filesystem.chmod(self.probe_path, 0o744)
+        self.assertTrue(filesystem.is_word_reachable(self.probe_path))
+
+        filesystem.chmod(self.probe_path, 0o700)
+        self.assertFalse(filesystem.is_word_reachable(self.probe_path))
```

Comment:
nit: Can we also create a test for a case where there is an ACE for an SID, but it's permissions are not enough? It's not a big deal, but that case doesn't have test coverage which seems easy to fix.

Reply:
Done.